TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/DriveSubsystem.java

```diff
@@ -30,8 +30,8 @@ public class DriveSubsystem extends SubsystemBase {
 
     private final BHI260IMU imu;
 
-    private double speedMultiplier = 1.0;
-    private boolean fieldCentric = true;
+    public double speedMultiplier = 1.0;
+    public boolean fieldCentric = true;
```

Comment:
These variables cannot be public, it goes against best practices. To access them from different class please write a getter.